apps/backend/src/app/common/interceptors/not-found.interceptor.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  BadRequestException,
+  CallHandler,
+  ExecutionContext,
+  Injectable,
+  InternalServerErrorException,
+  NestInterceptor,
+  NotFoundException,
+} from '@nestjs/common';
+import { catchError, Observable, throwError } from 'rxjs';
+import { EntityNotFoundError } from 'typeorm';
+
+@Injectable()
+export class NotFoundInterceptor implements NestInterceptor {
+  intercept(context: ExecutionContext, next: CallHandler): Observable<any> {
+    return next.handle().pipe(
+      catchError((err) => {
+        console.error(err);
```

Comment:
Instead of common `console.error()` it's wort to add fully-featured logger (https://docs.nestjs.com/techniques/logger)

Reply:
Done, used Nest Logger